src/structures/APIMessage.js

```diff
@@ -148,6 +149,12 @@ class APIMessage {
     }
     const embeds = embedLikes.map(e => new MessageEmbed(e).toJSON());
 
+    let components;
+    if (this.options.components) {
+      components = [];
+      components.push(...this.options.components.map(c => BaseMessageComponent.create(c).toJSON()));
```

Comment:
why not just `components = this.options.components.map(...);` ?

Reply:
good question, ask past souji

Reply:
wait, this is a monbrey PR... oops

Reply:
I guess it would have to be `this.options.components?.map` but otherwise yeah. We want it to remain undefined when not passed in